models/issue.go

```diff
@@ -1128,6 +1128,7 @@ type IssuesOptions struct {
 	PosterID           int64
 	MentionedID        int64
 	ReviewRequestedID  int64
+	SubscribedID       int64
```

Comment:
UserId? SubscriberId?

Reply:
It's just like the other IDs (`PosterID`, `MentionedID`...)

Reply:
When we say "an ID of a poster" or "an ID of the mentioned", it's clear the ID is for a user.

But when we say "an ID of the subscribed", it may refer to a subscribed issue, a little misleading. A user is a subscriber, only issues can be subscribed.

Reply:
Okay, thi s was your thought. I can change it.